packages/lesswrong/server/tagging/tagCallbacks.ts

```diff
@@ -155,11 +155,11 @@ export function voteUpdatePostDenormalizedTags({newDocument}: {newDocument: Vote
   void updatePostDenormalizedTags(postId);
 }
 
-export async function recomputeContributorScoresFor(votedRevision: DbRevision, vote: DbVote) {
+export async function recomputeContributorScoresFor(votedRevision: DbRevision, vote: DbVote, collectionName: CollectionNameString) {
   if (vote.collectionName !== "Revisions") return;
-  if (votedRevision.collectionName !== "Tags") return;
+  if (votedRevision.collectionName !== "Tags" && votedRevision.collectionName !== "MultiDocuments") return;
   
   const tag = await Tags.findOne({_id: votedRevision.documentId});
   if (!tag) return;
-  await updateDenormalizedContributorsList(tag);
+  await updateDenormalizedContributorsList({ document: tag, collectionName, fieldName: votedRevision.collectionName === "Tags" ? "description" : "contents" });
```

Comment:
Do we need to now fetch both the tag and the multidoc to make sure at least one of them exists (and pass in that one)?  Right now it looks like this just won't work for multidocs - I'm guessing it didn't matter because this function was only getting called in votingCallbacks.  But plausibly this also needs to get called when new revisions are created?  (Maybe that already happens because they get self-voted-on... need to confirm.)